chapters/VK_EXT_present_timing/queries.adoc

```diff
@@ -0,0 +1,476 @@
+associated flink:vkQueuePresentKHR commands.
+
+Because timing information for some present stages may: be available earlier
+than for some others, fname:vkGetPastPresentationTimingEXT may: return
+incomplete results, containing only information for a subset of the
+requested present stages. Further calls to
+fname:vkGetPastPresentationTimingEXT will keep providing all available
+results for a previously incomplete entry until it is complete.
+
+Upon return, zero or more slots of the pname:swapchain internal timing
+collection queue, equal to the number of completed entries written to
+pname:pPresentationTimings for which all present stages have been returned,
+are made available for future fname:vkQueuePresentKHR calls.
+
+include::{generated}/validity/protos/vkGetPastPresentationTimingEXT.adoc[]
+--
+
+[open,refpage='VkPastPresentationTimingEXT',desc='Structure containing timing information about a previously-presented image',type='structs']
+--
+
+The sname:VkPastPresentationTimingEXT structure is defined as:
+
+include::{generated}/api/structs/VkPastPresentationTimingEXT.adoc[]
+
+  * pname:sType is the type of this structure.
+  * pname:pNext is `NULL` or a pointer to a structure extending this
+    structure.
+  * pname:presentId is an application-provided value that was given to a
+    previous fname:vkQueuePresentKHR command via
+    slink:VkPresentIdKHR::pname:pPresentIds.
+    It can: be used to uniquely identify a previous present with the
+    flink:vkQueuePresentKHR command. If no slink:VkPresentIdKHR was provided
+    or if the specified present id value was zero, pname:presentId is zero.
+  * pname:targetPresentTime is an application-provided value that was given
+    to a previous flink:vkQueuePresentKHR command via
+    slink:VkPresentTimeEXT::pname:targetPresentTime.
+    If non-zero, it was used by the application to indicate that an image
+    not be presented any sooner than pname:targetPresentTime, relative to
+    the stage specified in
+    slink:VkSwapchainPresentTimingCreateInfoEXT::pname:presentStageTarget.
+    If the pname:swapchain is using relative present times, it is zero and
+    the actual present duration can: be obtained by comparing the
+    pname:pPresentStages results of consecutive presents.
+  * pname:presentStageCount is a count of items contained in
+    pname:pPresentStages.
+  * pname:pPresentStages is `NULL` or a pointer to an array of
+    slink:VkPresentStageTimeEXT providing timing information for each of the
+    present stages requested in
+    slink:VkSwapchainPresentTimingCreateInfoEXT::pname:presentStageQueries.
+  * pname:timingPropertiesChanged is ename:VK_TRUE if the swapchain's
+    timing properties have changed since the last time those
+    properties were queried with
+    flink:vkGetSwapchainTimingPropertiesEXT, otherwise the properties
+    have not changed.
+  * pname:timeDomainChanged is ename:VK_TRUE if the time domain enabled for
+    the swapchain is not currently available.
```

Comment:
The text for `timingPropertiesChanged` and `timeDomainChanged` suggest that their values are independent of the returned `presentId`. If that's true, then they should be relocated to a different api location. If they actually are dependent on `presentId`, then their text needs clarification.